spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -678,6 +655,202 @@ boolean result = pollableSource.poll(received -> {
+
+[source,java]
+----
+@StreamListener(“errorChannel")
+public void error(Message<?> message) {
+	System.out.println("Handling ERROR: " + message);
+}
+----
+
+This may be a convenient option if error handling logic is the same regardless of which handler produced the error.
+
+Also, error messages sent to the `errorChannel` can be published to a specific destination at the broker by configuring a binding named `error` for the outbound target.
+This option provides a mechanism to automatically send error messages to another application bound to that destination or for later retrieval (e.g., audit).
+For example, to publish error messages to a broker destination named `myErrors`, set the following property:
+
+[source,text]
+----
+spring.cloud.stream.bindings.error.destination=myErrors.
+----
+
+===== System Error Handling
+
+System level error handling implies that the errors are communicated back to the messaging system and given that not every messaging system
+is the same the capabilities may differ from binder to binder.
+
+That said, in this section we’ll explain the general idea behind system level error handling and will use Rabbit binder as an example. NOTE: Kafka binder provides similar
+support although some configuration properties do differ. Also for more details and confuration options please refer to individual binder's documentation.
+
+If no internal error handlers are configured the errors are propagated to the binders and binders subsequently propagate those errors back to the messaging system.
+Depending on the capabilities of the messaging systems such system may _drop_ the message, _re-queue_ the message for re-processing or _send the failed message to DLQ_.
+Both Rabbit and Kafka support these concepts. However other binders may not, so please refer to individual binder’s documentation for details on supported system
+level error handling options.
+
+====== Drop
+
+By default if no additional system level configuration is provided the messaging system will simply drop the failed message.
+While acceptable in some cases, for most cases it is not and we need some recovery mechanism to avoid message loss.
+
+====== DLQ - Dead Letter Queue
+
+DLQ allows failed messages to be send to a special destination - _Dead Letter Queue_.
+
+When configured, failed messages will be sent to this destination for subsequent re-processing or auditing/reconciliation.
+
+For example, continuing on the previous example and to set up the DLQ with Rabbit binder you need to set the following property
+
+[source,text]
+----
+spring.cloud.stream.rabbit.bindings.input.consumer.auto-bind-dlq=true
+----
+
+Keep in mind that in the above property the word `input` corresponds to the name of the input destination binding.
+The word `consumer` states that it is a consumer property and  `auto-bind-dlq` instructs the binder to configure DLQ for `input`
```

Comment:
Replace "The word `consumer` states" with "`consumer` indicates"
Add a comma after "property"
Remove the extra space before `auto-bind-dlq`